uber/templates/regform.html

```diff
@@ -75,15 +75,6 @@
 
     {% endif %}
 
-//    $(function () {
-//        staffingClicked();
-//        if ($.field('staffing')) {
-//            {% if c.AT_THE_CON %} $.field('staffing').prop('readonly', true).prop('title', 'Please see Staffing Operations to change your volunteer status.'); {% endif %}
-//            {% if attendee.shifts %} $.field('staffing').prop('readonly', true).prop('title', 'Please {{ "contact your department head to drop shifts" if c.AT_THE_CON else "unassign yourself from shifts" }} before changing your volunteer status.'); {% endif %}
-//            $.field('staffing').on('click', staffingClicked);
-//        }
-//    });
-
```

Comment:
If I remember correctly, this was from a quick fix at MAGStock? We can put this back in, or fix it proper in another pull request.